CourseGrab/ViewControllers/Home/HomeTableViewCell.swift

```diff
@@ -143,10 +143,11 @@ class HomeTableViewCell: UITableViewCell {
         
         popularityImageView.snp.makeConstraints { make in
             make.top.equalTo(courseLabel.snp.bottom).offset(9)
-            make.trailing.equalTo(popularityLabel.snp.leading).offset(-6)
+            make.trailing.equalTo(popularityLabel.snp.leading).inset(-4)
```

Comment:
You went from `offset(-6)` to `inset(-4)`. Is this what you mean to do? Did you not mean `offset(-6)` to `inset(6)` or `inset(4)?

Reply:
`inset(4)` actually causes the people image to overlap with the number representing the count of people tracking a course, so instead I use `inset(-4)`.